### ISSUE TITLE
the mailscanner does not count blocked messages in the web interface and thus the message counts are wrong

### ISSUE THREAD

**Issue by pdwalker** [mailwatch/1.2.0#249]
The following patch resolves an issue where blocked files which are also marked as spam do not get counted as "blocked files" in the MailWatch interface.

```php
[user@efa mailscanner]# diff -c functions.php.org functions.php
*** functions.php.org    2016-05-05 10:48:31.136301817 +0800
--- functions.php    2016-05-05 10:49:29.659234606 +0800
***************
*** 431,438 ****
      nameinfected>0
      AND (virusinfected=0 OR virusinfected IS NULL)
      AND (otherinfected=0 OR otherinfected IS NULL)
!     AND (isspam=0 OR isspam IS NULL)
!     AND (ishighspam=0 OR ishighspam IS NULL)
     THEN 1 ELSE 0 END
    ) AS blockedfiles,
    ROUND((
--- 431,438 ----
      nameinfected>0
      AND (virusinfected=0 OR virusinfected IS NULL)
      AND (otherinfected=0 OR otherinfected IS NULL)
!     -- AND (isspam=0 OR isspam IS NULL)
!     -- AND (ishighspam=0 OR ishighspam IS NULL)
     THEN 1 ELSE 0 END
    ) AS blockedfiles,
    ROUND((
***************
*** 441,448 ****
       nameinfected>0
       AND (virusinfected=0 OR virusinfected IS NULL)
       AND (otherinfected=0 OR otherinfected IS NULL)
!      AND (isspam=0 OR isspam IS NULL)
!      AND (ishighspam=0 OR ishighspam IS NULL)
      THEN 1 ELSE 0 END
     )/COUNT(*))*100,1
    ) AS blockedfilespercent,
--- 441,448 ----
       nameinfected>0
       AND (virusinfected=0 OR virusinfected IS NULL)
       AND (otherinfected=0 OR otherinfected IS NULL)
!      -- AND (isspam=0 OR isspam IS NULL)
!      -- AND (ishighspam=0 OR ishighspam IS NULL)
      THEN 1 ELSE 0 END
     )/COUNT(*))*100,1
    ) AS blockedfilespercent, 
```

**tldr;**
- blocked files that are spam are not counted as blocked files; all blocked files should be counted as blocked files
- spam that are blocked are not counted as spam; which is correct.

**Comment by endelwar** [comment]
fixed by 2c283b2a50cf758878df11bd9bf3cd840c7172b5
